WalletWasabi.Gui/Controls/WalletExplorer/WalletInfoViewModel.cs

```diff
@@ -29,7 +24,7 @@ public WalletInfoViewModel(WalletViewModel walletViewModel) : base(walletViewMod
 		{
 			ClearSensitiveData(true);
 			_warningMessage = "";
-			Closing = new CancellationTokenSource().DisposeWith(Disposables);
+			Closing = new CancellationTokenSource();
```

Comment:
CancellationTokenSource must be disposed somehow.

Reply:
ack